student-store-ui/src/components/Orders/Orders.jsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react";
+import "./Orders.css";
+import { useParams } from "react-router-dom";
```

Comment:
Can combine this line with the`useLocation` import, much like how you write `useEffect` and `useState`